pages/windows/dir.md

```diff
@@ -0,0 +1,19 @@
+# DIR 
+
+> Displays file and folders in current directory or given directory.
+
+- Show list of files and folder of current directory:
+
+`dir`
+
+- Show list of file and folder in any other directory:
+
+`dir {{path/to/<placeholder>}}`
+
+- Show hidden file and folders in current directory:
```

Comment:
Same comments as above. Please change files and folders uniformly to plural. And update the placeholder text.